### PR TITLE
Proxmox hyperconverged cloud tutorial

### PR DESCRIPTION
**Pull request by Kenzim**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Kenzi Marcel kenzi@kenzim.co.uk

### CODE REVIEW

File: tutorials/hyperconverged-proxmox-cloud/01.en.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+### DHCP Server for Internal NAT Network
+
+### Connecting to Hetzner Cloud 
+
+### High availability on ZeroTier Meshed VPN
+
+## Conclusion
+
+Time to get that 99.9999% uptime money
```
Comment (mhupfauer) [inline tutorials/hyperconverged-proxmox-cloud/01.en.md:227]: No content below your headings.
And "Time to get that 99.9999% uptime money" is probably a little to unprofessional.

File: tutorials/hyperconverged-proxmox-cloud/01.en.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+| Name               | Quantity | Cost (EUR) | Subtotal (EUR) |
+| ------------------ | -------- | ---------- | -------------- |
+| 12 port 10G switch | 1        | 43         | 43             |
+| 8 port 1G switch   | 12       | 2          | 2              |
+|                    |          | Total:     | €45            |
+
+For allowing servers to start with the same IP on any server, we will also need to purchase vSwitch IPs. These IPs can be used by any server at any time thats connected to the switch, as opposed to having IPs only usable on one server. This means your virtual machine can run on any node from the outside it would look the same. For 29 IPs, this will cost €35.29, which is probably the amount you'll need for a setup of this size. You can buy less if you know you are going to be using fewer, larger VMs instead of many smaller ones. Alternatively, if this is a private cloud, and you are using a meshed VPN like ZeroTier or Tinc, you won't need these external IPs.
+
+[//]: #write about reserving rackspace
```
Comment (mhupfauer) [inline tutorials/hyperconverged-proxmox-cloud/01.en.md:50]: does not render on hcloud community tutorial page

File: tutorials/hyperconverged-proxmox-cloud/01.en.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+Next, we need to add our servers to the vSwitch:
+
+<img src="images/vswitch3.png" alt="Creating vSwitch" style="zoom:67%;" />
+
+Finally, we can order some external IP addresses for the vSwitch:
+
+<img src="images/vswitch4.png" alt="Creating vSwitch" style="zoom:67%;" />
+
+[//]: #assess if images are really needed
```
Comment (mhupfauer) [inline tutorials/hyperconverged-proxmox-cloud/01.en.md:84]: see other comments.

File: tutorials/hyperconverged-proxmox-cloud/01.en.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+iface eno2 inet static
+      address 10.254.254.10
+      netmask 255.255.255.0
+```
+
+[//]: #find out the actual interface names for ex62 servers
+
+### Step 3.3 - Configuring vSwitch Network
+
+To utilise the vSwitch IPs, we will need to create a network for the VLan. Below is an example configuration:
```
Comment (mhupfauer) [inline tutorials/hyperconverged-proxmox-cloud/01.en.md:120]: VLAN

**Review by mhupfauer — COMMENTED** [review]
Some little comments in the tutorial.
The last headings do not contain any content, is this intentional or a issue?

File: tutorials/hyperconverged-proxmox-cloud/01.en.md (hunk 4 above)
Comment (Kenzim) [reply tutorials/hyperconverged-proxmox-cloud/01.en.md:120]: My bad this was an older commit that I didn't update properly.

**Review by mhupfauer — APPROVED** [review]

**Review by rook59 — COMMENTED** [review]
LGTM

The only thing missing is the cluster creation so that all proxmox servers can communicate with each other. Can you add it please?

**Comment by Kenzim** [comment]
> LGTM
> 
> The only thing missing is the cluster creation so that all proxmox servers can communicate with each other. Can you add it please?

good idea, I will add it when I can
thanks

**Comment by leojakeson** [comment]
@Kenzim Any update on the status of this? We'd like to merge this tutorial sooner rather than later, so it would be great if you could add that final part.

**Comment by Kenzim** [comment]
Sorry for the delay in adding this section, I've been bogged down with exams for the past few weeks. Let me know if there's anything else. Thanks

**Review by rook59 — APPROVED** [review]
Thanks!

**Pull request merged**